core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -84,6 +84,30 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       "-J", "-watchdog_client_retry_timeout", "-J", conf.clientCheckRetryTimeout.toString
     )
 
+    if (conf.jks.isDefined) {
+      cmdToLaunch ++ Seq("-jks", conf.jks.get)
+    }
+
+    if (conf.jksPass.isDefined) {
+      cmdToLaunch ++ Seq("-jks_pass", conf.jksPass.get)
+    }
+
+    if (conf.jksAlias.isDefined) {
+      cmdToLaunch ++ Seq("-jks_alias", conf.jksAlias.get)
```

Comment:
https://github.com/h2oai/h2o-3/pull/3905

Reply:
Yup, good point, we need to expose it also on internal backend ( or just check if it's already exposed there)